### ISSUE TITLE
DEV- Min worker count default help command text needs to be changed

### ISSUE THREAD

**Issue by shri-astro** [astronomer/astro-cli#1349]
The default value for the min worker count for a Celery deployment is 0 in the UI and in the CLI but the help command issued for default worker queue update indicates that the min worker count is 1.

```
shrividyahegde@Shrividyas-MacBook-Pro ~ % astro deployment inspect
Select a Deployment
 #     DEPLOYMENT NAME       RELEASE NAME                DEPLOYMENT ID                   DAG DEPLOY ENABLED     
 1     Kubernetes_shared     popular-rotation-3223       cll3hk62s1434731uymoywi92rr     false                  
 2     CE dedicated          prehistoric-axis-6302       cll3k5e1q2064611b1jkfis8z2a     false                  
 3     CE shared             siderean-supernova-8319     cll3k8j5n2039971u1unnf93h2r     false                  

> 2
deployment:
    configuration:
        name: CE dedicated
        description: ""
        runtime_version: 8.8.0
        dag_deploy_enabled: false
        ci_cd_enforcement: false
        scheduler_size: small
        is_high_availability: false
        executor: CeleryExecutor
        scheduler_au: 10
        scheduler_count: 1
        cluster_name: Dedicated cluster for deployment testing08:45:17.582125
        workspace_name: Astro_hosted_CLI_Automation_Restricted_Workspace
        deployment_type: HOSTED_DEDICATED
        cloud_provider: gcp
        region: us-central1
    worker_queues:
        - name: default
          max_worker_count: 10
          min_worker_count: 0
          worker_concurrency: 5
          worker_type: a5
    metadata:
        deployment_id: cll3k5e1q2064611b1jkfis8z2a
        workspace_id: clkmpjzfk002d01m8vnpg1tu7
        cluster_id: cll3hj39500th01mun6f6xc31
        release_name: N/A
        airflow_version: 2.6.3
        current_tag: 8.8.0
        status: HEALTHY
        created_at: 2023-08-09T09:58:37.406Z
        updated_at: 2023-08-09T10:15:25.16Z
        deployment_url: cloud.astronomer-dev.io/clkmpjzfk002d01m8vnpg1tu7/deployments/cll3k5e1q2064611b1jkfis8z2a/analytics
        webserver_url: org-shri-hosted-dedicated-1.astronomer-dev.run/dfis8z2a
        workload_identity: astro-prehistoric-axis-6302@hcdp-cll3hj39500th01mun6f6xc31.iam.gserviceaccount.com

shrividyahegde@Shrividyas-MacBook-Pro ~ % astro deployment worker-queue update default -h
Update a worker queue for an Astro Deployment

Current Context: Astro

Usage:
  astro deployment worker-queue update [flags]

Aliases:
  update, up

Flags:
      --concurrency int          The concurrency(number of slots) of the worker queue.
  -d, --deployment-id string     The deployment where the worker queue should be created.
      --deployment-name string   Name of the deployment where the worker queue should be created.
  -f, --force                    Force update: Don't prompt a user for confirmation
  -h, --help                     help for update
      --max-count int            The max worker count of the worker queue.
      --min-count int            The min worker count of the worker queue. (default -1)
  -n, --name string              The name of the worker queue. Queue names must not exceed 63 characters and contain only lowercase alphanumeric characters or '-' and start with an alphabetical character.
  -t, --worker-type string       The worker type of the worker queue.

Global Flags:
      --verbosity string      Log level (debug, info, warn, error, fatal, panic (default "warning")
      --workspace-id string   workspace assigned to deployment

shrividyahegde@Shrividyas-MacBook-Pro ~ % 

```

**What CLI Version did you experience this bug?**
1.18.1

**This CLI bug is related to which Astronomer Platform?**

- [ ] Astro

**What Operating System is the above CLI installed on?**
Mac M1 Pro

**Comment by sunkickr** [comment]
you mean it says -1 right?

**Comment by kushalmalani** [comment]
fix was released in 1.25.0
